pipelines/scipipe/ap_verify.groovy

```diff
@@ -438,16 +438,32 @@ def void runApVerify(Map p) {
 
       cd ${RUN_DIR}
       # -p ignored if not -g 3
+      status=0
+      extra_args=""
+      # deal with NAMESPACE and REST_PROXY_URL being empty strings
+      if [[ -n "$NAMESPACE" || -n "$REST_PROXY_URL" ]]; then
+        [[ -n "$NAMESPACE" ]] && extra_args+=" -n ${NAMESPACE}"
+        [[ -n "$REST_PROXY_URL" ]] && extra_args+=" -u ${REST_PROXY_URL}"
+      fi
```

Comment:
Shell script isn't my strong suit, but wouldn't it be simpler to have two independent `if` statements?

Reply:
Yes. Changed.